packages/libmusicbrainz/build.sh

```diff
@@ -2,17 +2,22 @@ TERMUX_PKG_HOMEPAGE=https://musicbrainz.org/doc/libmusicbrainz
 TERMUX_PKG_DESCRIPTION="The MusicBrainz Client Library"
 TERMUX_PKG_LICENSE="LGPL-2.1"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=()
+TERMUX_PKG_VERSION=(
+	5.1.0
+	2.9.12 # libxml2 version
+	0.32.1 # libneon version
+)
 TERMUX_PKG_REVISION=3
-TERMUX_PKG_VERSION+=(5.1.0)
-TERMUX_PKG_VERSION+=(2.9.12) # libxml2 version
-TERMUX_PKG_VERSION+=(0.32.1) # libneon version
-TERMUX_PKG_SRCURL=(https://github.com/metabrainz/libmusicbrainz/releases/download/release-${TERMUX_PKG_VERSION}/libmusicbrainz-${TERMUX_PKG_VERSION}.tar.gz
-                   ftp://xmlsoft.org/libxml2/libxml2-${TERMUX_PKG_VERSION[1]}.tar.gz
-                   https://notroj.github.io/neon/neon-${TERMUX_PKG_VERSION[2]}.tar.gz)
-TERMUX_PKG_SHA256=(6749259e89bbb273f3f5ad7acdffb7c47a2cf8fcaeab4c4695484cef5f4c6b46
-                   c8d6681e38c56f172892c85ddc0852e1fd4b53b4209e7f4ebf17f7e2eae71d92
-                   05c54bc115030c89e463a4fb28d3a3f8215879528ba5ca70d676d3d21bf3af52)
+TERMUX_PKG_SRCURL=(
+	https://github.com/metabrainz/libmusicbrainz/releases/download/release-${TERMUX_PKG_VERSION}/libmusicbrainz-${TERMUX_PKG_VERSION}.tar.gz
```

Comment:
I can hear Shellcheck yelling at you.
```suggestion
	https://github.com/metabrainz/libmusicbrainz/releases/download/release-${TERMUX_PKG_VERSION[0]}/libmusicbrainz-${TERMUX_PKG_VERSION[0]}.tar.gz
```